src/containers/shared/components/Transaction/XChainModifyBridge/Simple.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Amount } from '../../Amount'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { XChainBridge } from '../XChainBridge'
+
+export const Simple: TransactionSimpleComponent = (
+  props: TransactionSimpleProps,
+) => {
+  const { t } = useTranslation()
+  const {
+    data: {
+      instructions: {
+        lockingDoor,
+        lockingIssue,
+        issuingDoor,
+        issuingIssue,
+        signatureReward,
+        minAccountCreateAmount,
+        bridgeOwner,
+      },
+    },
+  } = props
+
+  return (
+    <>
+      <XChainBridge
+        lockingDoor={lockingDoor}
+        lockingIssue={lockingIssue}
+        issuingDoor={issuingDoor}
+        issuingIssue={issuingIssue}
+        signatureReward={signatureReward}
+        bridgeOwner={bridgeOwner}
+      />
+      {minAccountCreateAmount && (
```

Comment:
Can you add a test for when this would be hit?  More than the test having a transaction json for the scenario will be valuable.